pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -202,6 +202,22 @@ flux reconcile -n kommander-flux source git management --kubeconfig MANAGED_KUBE
 ✔ fetched revision main/GIT_HASH
 ```
 
+### Cert-manager expiration workaround
+
+`cert-manager` renews all certificates 60 days after you install Kommander on your cluster. Unfortunately, some applications or pods fail to receive the renewed certificate information, causing them to break upon expiration (90 days after Kommander was installed, which normally coincides with the date you created the cluster). As a result, your cluster stops running, and sometimes, you are unable to access the UI.
```

Comment:
We have a rule in support that we *never* use the word 'unfortunately' in customer facing communications, and I don't think we should use it in documentation either.  

We should be more straightforward here about the fact that the 2.1.0 and 2.1.1 Kommander installation just doesn't handle certificate renewal properly - this current phrasing kind of implies it just 'happened'.    In my experience, documentation is better received by technical audiences if you just explain the problem directly rather than trying to beat around the bush.   

```suggestion
Due to an oversight, Kommander 2.1.0 and 2.1.1 installations do not properly handle certificate renewal for kommander applications.  The`cert-manager` component renews all certificates 60 days after you install Kommander on your cluster.  When this occurs, some of the kommander applications and pods fail to receive the renewed certificate information, causing them to stop working upon expiration.  This occurs 90 days after Kommander was installed, which normally would coincide with the date you created the cluster.  While the effects can vary, the most common failure is the inability to login to the UI due to an expired certificate in the dex-k8s-authenticator pod.
```